meta/FSMathLink.m

```diff
@@ -303,6 +369,39 @@
     TextFormatting`IndentText["MLPut(link, id);\n"] <> "}\n" <>
     "MLPut(link, " <> decayName <> ".get_width());\n";
 
+PutEffCTableEntries[modelName_] :=
+    Module[{body = "", particleList = {}},
+       particleList = {{-1, 1, "dd"}, {-2, 2, "uu"}, {-3, 3, "ss"}, {-4, 4, "cc"}, {-5, 5, "bb"}, {-6, 6, "tt"}, {-11, 11, "ee"}, {-13, 13, "mumu"}, {-15, 15, "tautau"}, {-24, 24, "WW"}, {23, 23, "ZZ"}, {22, 23, "Zgam"}, {21, 21, "gg"}, {21, 22, "gamgam"}};
```

Comment:
My proposal to create a local array of `std::pair<int,int>` could be used here: The `particleList` could be removed and one could access the struct of pairs. But then it may be best to put the array of pairs where it is globally accessible, maybe in `src/`.